modules/dbnd/src/dbnd/_core/task_run/task_run_sync_local.py

```diff
@@ -50,11 +51,9 @@ def __init__(self, task_run):
 
     def _local_cache_target(self, target_):
         # type: (FileTarget) -> FileTarget
-        return target(
-            self.task_run.attemp_folder_local_cache,
-            os.path.basename(target_.path),
-            config=target_.config,
-        )
+        _, path = os.path.splitdrive(target_.path)
```

Comment:
Not sure that we have any use with `splitdrive`, we are not supporting windows, and on unix systems this doesn't change the path whatsoever

Reply:
remote path can has something like `gs://` or `s3://` am I right?

Reply:
oh wait `splitdrive` doesn't do what I expect it to

Reply:
Doesn't matter, it's a windows method. Here's my output:
```
>>> a = "s3://bucket/item"
>>> os.path.splitdrive(a)
('', 's3://bucket/item')
>>>
```

Reply:
```
     path name                         drive                  tail
On Windows
If path contains drive letter
C:\User\Documents\file.txt               C:           C:\User\Documents\file.txt

If the path contains UNC path 
\\host\computer\dir\file.txt       \\host\computer          \dir\file.txt

On Unix
/home/User/Documents/file.txt         {empty}        /home/User/Documents/file.txt   
```

Reply:
eh :( I'll think of something else then

Reply:
i pushed alternative solution